index.html

```diff
@@ -54,16 +54,225 @@ <h1 class="subSentenceText">I'm a Software Developer</h1>
+            <p class="textSegment">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="builtWith">
+              <li>
+                <button class="buttonOfTechStack">HTML</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Bootstrap</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Ruby</button>
+              </li>
+            </ul>
+            <button class="secondProjectButton">See Project</button>
+          </div>
+
+          <div class="Screen3">
+            <h3 class="PrintingData">Professional Art Printing Data</h3>
+            <p class="textSegment">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="builtWith">
+              <li>
+                <button class="buttonOfTechStack">HTML</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Bootstrap</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Ruby</button>
+              </li>
+            </ul>
+            <button class="secondProjectButton">See Project</button>
+          </div>
+
+          <div class="Screen4">
+            <h3 class="PrintingData">Professional Art Printing Data</h3>
+            <p class="textSegment">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="builtWith">
+              <li>
+                <button class="buttonOfTechStack">HTML</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Bootstrap</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Ruby</button>
+              </li>
+            </ul>
+            <button class="secondProjectButton">See Project</button>
+          </div>
+
+          <div class="Screen5">
+            <h3 class="PrintingData">Professional Art Printing Data</h3>
+            <p class="textSegment">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="builtWith">
+              <li>
+                <button class="buttonOfTechStack">HTML</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Bootstrap</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Ruby</button>
+              </li>
+            </ul>
+            <button class="secondProjectButton">See Project</button>
```

Comment:
- [ ] Nicw work so far 👍🏿 implement your items but I kindly suggest you remove the `button` elements and use only the `ul`` li `tags to lay out your items 😉

Reply:
- [ ]  Kindly let's implement the suggestion from the previous reviewer. 🙏🏽 It would enhance the quality of your code 😁 😁  You can achieve that by following the instruction I mentioned above ☝🏽 ☝🏽

Reply:
The previous reviewer suggested  you make two changes:
  - Remove the button elements from the code.
  - Use only the ul and li tags to lay out the items.
These changes will improve the quality of the code. You can follow these instructions:arrow_down:  by @rica213  to be able to implement the changes.

> * [ ]  Kindly let's implement the suggestion from the previous reviewer. 🙏🏽 It would enhance the quality of your code blush  You can achieve that by following the instruction below 👇🏽
> 
> 
> HTML Structure without the `button` 👇🏽
> 
> ![Screenshot from 2023-06-03 22-14-14](https://user-images.githubusercontent.com/10439283/243126724-20d43b6a-95e5-4d1b-b0d8-82a653ae7039.png)
> 
> In your CSS, you can use the CSS Selectors to target your `li`
> 
> ```shell
> .techStacks li {
>   font-weight: 400;
>   font-size: 15px;
>   text-align: center;
>   text-decoration: none;
>   display: inline-block;
>   background-color: #fff;
>   color: #091e42;
>   border: 2px solid #8993a4;
>   padding: 10px 12px;
>   margin-top: 12px;
> }
> ```

Here are the instructions in more detail:
    - To remove the button elements, find all instances of the button tag and delete them.
    - To use only the ul and li tags to lay out the items, create a new ul element and then add li elements to it for each item.